manifests/init.pp

```diff
@@ -29,11 +32,14 @@
   $target_dir   = 'UNDEF',
   $command_name = 'UNDEF',
   $user         = 'UNDEF',
+  $group        = 'UNDEF',
```

Comment:
please use for new params the undef constant. In a new major release I'll fix the legacy params to use the undef parameter, too